cobalt/network/disk_cache/cobalt_backend_impl.h

```diff
@@ -78,13 +78,14 @@ class NET_EXPORT_PRIVATE CobaltBackendImpl final
   std::unique_ptr<Iterator> CreateIterator() override;
   void GetStats(base::StringPairs* stats) override {}
   void OnExternalCacheHit(const std::string& key) override;
-  size_t DumpMemoryStats(
-      base::trace_event::ProcessMemoryDump* pmd,
-      const std::string& parent_absolute_name) const override;
+  //   size_t DumpMemoryStats(
```

Comment:
I have it commented out because we're not able to use most of base::trace_event (i.e. ProcesMemoryDump isn't defined here), but I'm not sure whether or not we'll need this function of something like it when we implement the class correctly

Reply:
Created b/326662008 to follow up on this (as well as other things here)